src/routes/categories/[topicSlug]/[tableSlug]/[categorySlug].svelte

```diff
@@ -0,0 +1,153 @@
+      locationId = $page.query.get("location");
+      locationName = getLadName(locationId);
+    }
+  }
+
+  $: appIsInitialised, $appIsInitialised && initialisePage();
+
+  function initialisePage() {
+    if (locationId) {
+      locationName = getLadName(locationId);
+      updateSelectedGeography(locationId);
+    }
+  }
+
+  $: innerWidth = 0;
+</script>
+
+<svelte:window bind:innerWidth />
+<svelte:head>
+  <title>2021 Census Data Atlas Categories</title>
+</svelte:head>
+
+<BasePage mobileMap={false} withoutBackground>
+  <span slot="header">
+    <Header
+      showBackLink
+      serviceTitle="Choose a data option {locationId ? `for ${locationName}` : ''}"
+      description="Choose a category and select an option within it to explore {locationName
+        ? `${locationName}'s`
+        : 'Census'} data."
+    />
+  </span>
+
+  <span slot="map">
+    <Map maxzoom={14}>
+      <TileSet
+        id="lad"
+        type="vector"
+        url={config.legacy.ladvector.url}
+        layer={config.legacy.ladvector.layer}
+        promoteId={config.legacy.ladvector.code}
+      >
+        {#if $categoryDataIsLoaded}
+          <DataLayer id="lad-data-zoom" data={categoryData} maxzoom={config.ux.map.lsoa_breakpoint} />
+        {/if}
+        <InteractiveLayer
+          id="lad-interactive-layer"
+          selected={$selectedGeography.lad}
+          maxzoom={config.ux.map.buildings_breakpoint}
+          onSelect={(code) => {
+            updateSelectedGeography(code);
+          }}
+          onHover={(code) => {
+            updateHoveredGeography(code);
+          }}
+          filter={config.ux.map.filter}
+        />
+      </TileSet>
+
+      <TileSet
+        id="lsoa"
+        type="vector"
+        url={config.legacy.lsoabounds.url}
+        layer={config.legacy.lsoabounds.layer}
+        promoteId={config.legacy.lsoabounds.code}
+        minzoom={config.ux.map.lsoa_breakpoint}
+        maxzoom={config.ux.map.buildings_breakpoint}
+      >
+        {#if $categoryDataIsLoaded}
+          <DataLayer id="lsoa-data" data={categoryData} />
+        {/if}
+      </TileSet>
+      <TileSet
+        id="lsoa-building"
+        type="vector"
+        url={config.legacy.lsoabldg.url}
+        layer={config.legacy.lsoabldg.layer}
+        promoteId={config.legacy.lsoabldg.code}
+        minzoom={config.ux.map.buildings_breakpoint}
+      >
+        {#if $categoryDataIsLoaded}
+          <DataLayer id="lsoa-data-zoom" data={categoryData} />
+        {/if}
+      </TileSet>
+      <TileSet
+        id="lad-boundaries"
+        type="vector"
+        url={config.legacy.ladvector.url}
+        layer={config.legacy.ladvector.layer}
+        promoteId={config.legacy.ladvector.code}
+      >
+        <BoundaryLayer minzoom={config.ux.map.lsoa_breakpoint} id="lad-boundary-layer" />
+      </TileSet>
+    </Map>
+  </span>
+
+  <TopicExplorer {locationId} selectedTopic={topicSlug} visitedTable={tableSlug} />
+
+  {#if innerWidth >= 500}
```

Comment:
this is a rendering-style breakpoint, right? Feels like this should be in a config file somewhere rather than hardcoded here?

Reply:
Done, I moved the breakpoint in the config file.